core/trino-main/src/main/resources/webapp/src/components/QueryHeader.jsx

```diff
@@ -103,6 +103,8 @@ export class QueryHeader extends React.Component {
                                     &nbsp;
                                     <a href={"/ui/api/query/" + query.queryId + "?pretty"} className="btn btn-info navbar-btn" target="_blank">JSON</a>
                                     &nbsp;
+                                    <a href={"/ui/api/query/" + query.queryId + "/explain"} className="btn btn-info navbar-btn" target="_blank">Explain</a>
```

Comment:
i guess opening in a new tab is convention so far in the UI .. but typically I am not a fan of that.